.cargo/config.toml

```diff
@@ -0,0 +1,3 @@
+[env]
+# If we're running in the project workspace, then we should set the workspace env var so we read/write to/from files relative to the workspace.
+CARGO_WORKSPACE_DIR = { value = "", relative = true }
```

Comment:
I'm skeptical about this - if it works, it doesn't follow the principle of least surprise, and I don't even know if it works...

Reply:
It [works](https://github.com/rust-lang/cargo/issues/3946#issuecomment-973132993), but I think it's fair to treat this as a bit unstable.